code/game/gamemodes/cult/cult_items.dm

```diff
@@ -72,7 +72,7 @@
 	item_state = "magus"
 	desc = "A helm worn by the followers of Nar-Sie."
 	flags = FPRINT
-	body_parts_covered = FULL_HEAD
+	body_parts_covered = FULL_HEAD|BEARD
```

Comment:
It would be, but I just removed it because lots of stuff compare your clothes to their coverage of full_head and that causes problems if they don't cover the beard